### PR TITLE
Login user

### PR DESCRIPTION
**Pull request by at-cuclt**
@at-dungnh I've finished login and logout user, please help me review it, many thanks :))

### CODE REVIEW

File: resources/views/frontend/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @include('frontend.layouts.partials._navbar')
+@endsection
+
+@section('content')
+<div class="row">
+    <!-- log in -->
+    <div class="col-md-4 col-md-offset-4">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title text-center" id="exampleModalLabel"><span style="font-size:30px; color:#2ecc71;">{!! trans('labels.login_title') !!}</span></h4>
```
Comment (dzungnguyenh) [inline resources/views/frontend/auth/login.blade.php]: [rank B] please define id more clearly and avoid to use css inline

File: resources/views/frontend/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,179 @@
+@extends('frontend.layouts.master')
```
Comment (dzungnguyenh) [inline resources/views/frontend/index.blade.php:1]: is this sample view only?

File: resources/views/frontend/layouts/partials/_navbar.blade.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<!-- Navigation -->
+    <nav class="navbar navbar-inverse navbar-fixed-top maumenu" role="navigation">
+        <div class="container">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/_navbar.blade.php]: [rank C] put this text to language file, please.

File: tatus [hunk 4]
```diff
@@ -0,0 +1,1872 @@
+[33mcommit fe06612e07112a49a75ff7f9cbf3534314a9209c[m
```
Comment (dzungnguyenh) [inline tatus]: [rank A] why do you want to commit this file?

File: resources/views/frontend/index.blade.php (hunk 2 above)
Comment (at-cuclt) [inline resources/views/frontend/index.blade.php]: I don't understand your mean.

Comment (at-cuclt) [inline resources/views/frontend/index.blade.php:1]: Yes.

File: resources/views/frontend/layouts/partials/_footer.blade.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<!-- Footer -->
+<footer>
+    <div class="bar">
+        <div class="bar-wrap">
+            <ul class="links"> <!-- footer menu -->
+                <li>
+                    <a href="#" title="Home">Home</a>
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/_footer.blade.php]: [rank C] Why don't they define in language file?

**Comment by dzungnguyenh** [comment]
LGTM. Thanks.

**Pull request merged**